src/rpc/mining.cpp

```diff
@@ -98,13 +98,12 @@ UniValue getnetworkhashps(const JSONRPCRequest& request)
 UniValue generateBlocks(std::shared_ptr<CReserveScript> coinbaseScript, int nGenerate, uint64_t nMaxTries, bool keepScript)
 {
     static const int nInnerLoopCount = 0x10000;
-    int nHeightStart = 0;
     int nHeightEnd = 0;
     int nHeight = 0;
 
     {   // Don't keep cs_main locked
         LOCK(cs_main);
-        nHeightStart = chainActive.Height();
+        int nHeightStart = chainActive.Height();
```

Comment:
is this var even needed? could do `nHeight = chainActive.Height();` and use `nHeight` instead of `nHeightStart` on the line after.